build.sbt

```diff
@@ -42,6 +42,7 @@ lazy val root = (project in file(".")).
         , "org.json4s" %% "json4s-core" % "3.2.10"
         , "org.json4s" %% "json4s-jackson" % "3.2.10"
         , "org.json4s" %% "json4s-ast" % "3.2.10"
+        , "org.elasticsearch" % "elasticsearch-spark-20_2.11" % "6.2.4"
```

Comment:
I use sbt often (it's faster) and this was added to build.gradle recently, so I added it to build.sbt while I was working.